solution.rb

```diff
@@ -35,6 +35,13 @@
     #   end
     #   p new_arr
     # ```
+
+    students.each do |i|
```

Comment:
Great work, but the variable's name represents what information the variable contains so change the variable name from ```i``` to ```student```
``` students.each do |student|```